pkg/requests/ids.go

```diff
@@ -0,0 +1,51 @@
+package requests
+
+import "regexp"
+
+var idURLMap = map[string]string{
+	"txn":   "/v1/balance_transactions/",
+	"ch":    "/v1/charges/",
```

Comment:
Can you add an entry for `py` that also points to `/v1/charges`? (Some charge objects have a `py_` prefix because Reasons™)

Reply:
😬 yep!